product_docs/docs/pgd/5/admin-tpa/upgrading_major_rolling.mdx

```diff
@@ -94,13 +94,14 @@ SSH into kaboom and use the following command to connect to PGD's `bdrdb` databa
 ```bash
 sudo -u postgres psql bdrdb
 ```
-Then, when connected to bdrdb, use the following query:
+
+When connected to bdrdb, use the following query:
```

Comment:
Don't know why I can't edit, but bdrdb should be tagged.

Reply:
I fixed it.